build-tools/packages/build-cli/docs/typetestDetails.md

```diff
@@ -79,7 +79,7 @@ on a particular branch. Baseline versions are set as follows based on the branch
 | Branch release designation | Baseline version | Example: version 2.3.4 |
 | -------------------------- | ---------------- | ---------------------- |
 | `patch`                    | `previousPatch`  | **2.3.3**              |
-| `minor`                    | `^previousMinor` | **^2.2.0**             |
+| `minor`                    | `~previousMinor` | **~2.2.0**             |
```

Comment:
I think `~` over `^` is correct for minor, and hope that's what the code does?

Reply:
Yes, you're right. Relevant code is here:

https://github.com/microsoft/FluidFramework/blob/586bd7fc631d55bbb461e7c15fa0fb6a51897023/build-tools/packages/build-tools/src/typeValidator/packageJson.ts#L390-L396

The original feedback during the design review was to use `^`, but during development I became convinced that `~` is more correct for this purpose, for the reasons you noted - it's confusing that a previous version would use a range that could include the version being tested against. :smile:

I left other combinations in place even if we don't use them with the rationale that we don't know enough about our needs at the moment. We might need it. Ordinarily I think this is bad logic, but in this case, I think it's appropriate.